src/taxes/views/ChannelsList.tsx

```diff
@@ -1,22 +1,71 @@
+import {
+  useTaxConfigurationsListQuery,
+  useTaxConfigurationUpdateMutation
+} from "@saleor/graphql";
 import useNavigator from "@saleor/hooks/useNavigator";
+import useNotifier from "@saleor/hooks/useNotifier";
+import useShop from "@saleor/hooks/useShop";
+import { commonMessages } from "@saleor/intl";
+import createDialogActionHandlers from "@saleor/utils/handlers/dialogActionHandlers";
+import { mapEdgesToItems } from "@saleor/utils/maps";
 import React from "react";
+import { useIntl } from "react-intl";
 
-import { taxConfigurations } from "../fixtures";
 import TaxChannelsPage from "../pages/TaxChannelsPage";
-import { channelsListUrl, taxTabSectionUrl } from "../urls";
+import {
+  channelsListUrl,
+  TaxesUrlDialog,
+  TaxesUrlQueryParams,
+  TaxTab,
+  taxTabPath
+} from "../urls";
 import { useTaxUrlRedirect } from "../utils/useTaxUrlRedirect";
 
 interface ChannelsListProps {
   id: string | undefined;
+  params: TaxesUrlQueryParams | undefined;
 }
 
-export const ChannelsList: React.FC<ChannelsListProps> = ({ id }) => {
+export const ChannelsList: React.FC<ChannelsListProps> = ({ id, params }) => {
   const navigate = useNavigator();
+  const notify = useNotifier();
+  const intl = useIntl();
 
-  const handleTabChange = (tab: string) => {
-    navigate(taxTabSectionUrl(tab));
+  const handleTabChange = (tab: TaxTab) => {
+    navigate(taxTabPath(tab));
   };
 
+  const [
+    taxConfigurationUpdateMutation,
+    { status: mutationStatus, loading: mutationInProgress }
+  ] = useTaxConfigurationUpdateMutation({
+    onCompleted: data => {
+      const errors = data?.taxConfigurationUpdate?.errors;
+      if (errors.length === 0) {
+        notify({
+          status: "success",
+          text: intl.formatMessage(commonMessages.savedChanges)
+        });
+      } else {
+        notify({
+          status: "error",
+          text: intl.formatMessage(commonMessages.defaultErrorTitle)
+        });
+      }
```

Comment:
I think it's handled by default in `makeMutation`

Reply:
It should be